spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/support/RateLimitEvent.java

```diff
@@ -10,7 +10,7 @@
  *
  * @author vasilaio
  */
-public class RateLimitEvent extends ApplicationEvent {
+public final class RateLimitEvent extends ApplicationEvent {
```

Comment:
Sorry for being picky, but I think `RateLimitExceededEvent` sounds better. I can change it myself if you don't want to!